pt-BR/mail/chrome/messenger/phishing-afterload-warning-message.dtd

```diff
@@ -1,28 +1,28 @@
-<!ENTITY safeb.palm.warning.title "Suspected Web Forgery">
+<!ENTITY safeb.palm.warning.title "Suspeita de falsificação na Internet">
 
-<!ENTITY safeb.palm.message.p1 "This page has been reported as a web forgery designed to trick users into sharing personal or financial information. Entering any personal information on this page may result in identity theft or other fraud. &#160;">
-<!ENTITY safeb.palm.message.p1.linkText "Read more &#187;">
-<!ENTITY safeb.palm.p1.linkStatusText "Read more &#133;">
+<!ENTITY safeb.palm.message.p1 "Esta página foi reportada como falsificação projetada para enganar os usuários e fazer com que forneçam dados pessoais ou informações financeiras. Digitar informações pessoais nesta página pode resultar em roubo de identidade ou outros tipos de fraude. &#160;">
+<!ENTITY safeb.palm.message.p1.linkText "Leia mais em &#187;">
+<!ENTITY safeb.palm.p1.linkStatusText "Leia mais em &#133;">
 
-<!ENTITY safeb.palm.message.p2.start "These types of web forgeries are used in scams known as phishing attacks, in which fraudulent web pages and emails are used to imitate sources you may trust. You can find out more about ">
-<!ENTITY safeb.palm.message.p2.linkText "how &brandShortName; protects you">
-<!ENTITY safeb.palm.message.p2.end " from phishing attacks.">
+<!ENTITY safeb.palm.message.p2.start "Esses tipos de falsificações são utilizados em golpes conhecidos como ataques de phishing, em que páginas de Internet falsas e e-mails são utilizados para imitar fontes nas quais você confia. Você pode saber mais sobre ">
+<!ENTITY safeb.palm.message.p2.linkText "como o &brandShortName; te protege">
+<!ENTITY safeb.palm.message.p2.end " de ataques de phishing.">
 
-<!ENTITY safeb.palm.accept.label "Get me out of here!">
-<!ENTITY safeb.palm.accept.statustext "Navigate to my home page">
-<!ENTITY safeb.palm.decline.label "Ignore this warning">
-<!ENTITY safeb.palm.decline.statustext "Close warning" >
-<!ENTITY safeb.palm.notforgery.label2 "This isn't a web forgery…">
-<!ENTITY safeb.palm.report.label "Why was this site blocked?">
+<!ENTITY safeb.palm.accept.label "Me tire daqui!">
+<!ENTITY safeb.palm.accept.statustext "Navegar para minha página inicial">
+<!ENTITY safeb.palm.decline.label "Ignorar este aviso">
+<!ENTITY safeb.palm.decline.statustext "Fechar aviso" >
+<!ENTITY safeb.palm.notforgery.label2 "Não é uma falsificação…">
+<!ENTITY safeb.palm.report.label "Por que este site foi bloqueado?">
```

Comment:
I disagree. "Por que" has no accent in the middle of a question, only at the end.